modules/src/movie.py

```diff
@@ -1,22 +1,46 @@
 import requests
 import requests_cache
 
+import config
 from templates.button import *
 
 
 def process(input, entities):
     output = {}
     try:
         movie = entities['movie'][0]['value']
+
+        # Make a search request to the API to get the movie's TMDB ID.
+        # TMDB required attribution notice: "This product uses the TMDb API but is not
+        # endorsed or certified by TMDb."
+
         with requests_cache.enabled('movie_cache', backend='sqlite', expire_after=86400):
-            r = requests.get('http://www.omdbapi.com/?t=' + movie + '&plot=full&r=json')
+            request_str = 'https://api.themoviedb.org/3/search/movie?api_key='
+            request_str += config.TMDB_API_KEY + '&query=' + movie
```

Comment:
It'd be cleaner to use `params` parameter in `requests.get()` method to generate the query string. You can refer to the `lyrics` module: https://github.com/swapagarwal/JARVIS-on-Messenger/blob/master/modules/src/lyrics.py#L17-L20